apps/ui/src/composables/useWeb3.ts

```diff
@@ -56,7 +57,7 @@ export function useWeb3() {
         });
         auth.provider.value.on('accountsChanged', async accounts => {
           if (accounts.length !== 0) {
-            state.account = accounts[0];
+            state.account = formatAddress(state.account);
```

Comment:
Why are we not using the new accounts list after `accountsChanged` ?

Reply:
Great catch, thanks!